packages/ui/src/components/Modal/style.less

```diff
@@ -1,5 +1,6 @@
 @import "antd/lib/modal/style/index.less";
 @import (reference) "@open-condo/ui/src/tokens/variables.less";
+@import "@open-condo/ui/src/components/style/core/global.less";
```

Comment:
inside `src/index.tsx` global styles are already imported, so you can just use it here

Reply:
ok